### DISCUSSION TITLE
How to change language

### DISCUSSION THREAD

**Discussion by marferre63** [sepinf-inc/IPED#135] [Q&A] [answered]
Machine translated:
I would like to know how to configure IPED version 3.17.2 so that it has the language and date settings in Portuguese.
I had been running data obtained in breach of telematic confidentiality with previous versions and the query tool was in Portuguese and the dates in the body of the emails in our dd / mm / yyyy format.
However in version 3.17 the tool is in English and the dates in the American format mm / dd / yyyy.

**Comment by lfcnassif** [comment, marked as answer]
Just change 'locale' option in LocalConfig.txt file. Please note currently the locale can't be changed after processing the case, see #18

**Comment by lfcnassif** [comment]
Please note currently we just have a few parsers for cellphone artifacts. Now iped supports dbx, mbox, pst, ost, msg and eml emails.

If you want other formats, you would be better processing them in Cellebrite software and importing the resulting ufdr file.

**Comment by marferre63** [comment]
Nassif,

Thanks for answered.

I've changed the localization/language to pt-BR (locale = pt-BR) but the IPED 3.17.2 don't start. 
I'm using a bat file with this command: 
d:\iped-3.17.2\iped-3.17.2\jre\bin\java -jar d:\iped-3.17.2\iped-3.17.2\iped.jar -Xmx=12GB -d "d:\TesteEmail" -o "d:\e-mail_teste_indexado"

In the 'TesteEmail' directory, I'm using a mbox file to test.

Returning the language to 'en', the IPED start normally.

**Comment by lfcnassif** [comment]
Did you read and follow the instructions printed in terminal when you changed locale to pt-BR?

**Comment by marferre63** [comment]
Thanks again

As I was using a bat file, I couldn't see the instructions about profile directory.

After set the language to pt-BR at LocalConfig.txt, I restored the IPED config files at root directory to the default and modified them at profile directory. With that, the indexing worked well. And the date format is showed as the brazilian.

From now on, I'll use the command line, in order to see the further instruccions, if somethig occour.

**Comment by lfcnassif** [comment]
You're welcome. Yes, important errors are printed to console, I suggest always looking at it.
